app/build.gradle

```diff
@@ -4,6 +4,9 @@ plugins {
     id 'com.google.android.libraries.mapsplatform.secrets-gradle-plugin'
 }
 
+def Properties properties = new Properties()
```

Comment:
use the current approach of the project for getting local.properties data, like the weatherAPIkey.